imageprocessor/tagservice/test.py

```diff
@@ -60,10 +60,17 @@ def test_detection_for_every_image(self):
                 self.assertTrue(len(tags) >= 1)
     def test_results_page_shows_image(self):
         client = Client()
-        with open(TEST_IMAGES_DIR + "/image1.jpg", "rb") as file:
-            response = client.post(reverse('classify'), { 'file' : file})
+        with open(TEST_IMAGES_DIR + "/image1.jpg", "rb") as file, open(TEST_IMAGES_DIR + "/image2.jpg", "rb") as file2:
+            response = client.post(reverse('classify'), { 'file' : [file, file2]})
         self.assertIsNotNone(response.context['new_image'])
 
+    def test_results_page_shows_tags(self):
+        client = Client()
+        with open(TEST_IMAGES_DIR + "/image1.jpg", "rb") as file, open(TEST_IMAGES_DIR + "/image2.jpg", "rb") as file2:
+            response = client.post(reverse('classify'), { 'file' : [file, file2]})
+        self.assertIsNotNone(response.context['tags'])
```

Comment:
It would be better to test the content of what's in the tags. Then again, we seem to have written nearly identical tests in our PRs so it probably doesn't matter